.build/build-owasp.xml

```diff
@@ -83,4 +85,67 @@
             </fileset>
         </dependency-check>
     </target>
+
+    <target depends="init" name="license-report">
+        <exec executable="jq" osfamily="unix" dir="${basedir}" logError="true" failonerror="false" failifexecutionfails="false">
+            <arg value="-M"/>
+            <arg value=".dependencies[] | .fileName + &quot;=&quot; +  .license"/>
+            <arg value="${build.dir}/owasp/dependency-check-report.json"/>
+            <redirector output="${build.dir}/owasp/license-report.txt"/>
+        </exec>
+
+        <loadfile property="wrongly-licensed" srcFile="${build.dir}/owasp/license-report.txt">
+            <filterchain>
```

Comment:
Here we have defined patterns for allowed licenses

Reply:
nice.

we should add a comment pointing to https://apache.org/legal/resolved